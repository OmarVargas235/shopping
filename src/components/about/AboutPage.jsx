import React from 'react';
import styled from 'styled-components';
import { ReactComponent as GitHubIcon } from '../../assets/iconos/git-hub.svg';

const Button = styled.button`
	padding: 12px 24px;
    border-radius: 0;
    background-color: #1a1a1a;
    color: #fff;
    font-size: .8rem;

    :hover {
		background-color: #070707;
	}
`;

const AboutPage = () => (
	<div className="text-center d-flex flex-column align-items-center container">
		<h2 className="font-normal">ABOUT</h2>

		<p className="my-4">This project was built for practice purposes using Context API by React.</p>

		<Button className="btn d-flex text-center align-items-center"> 
		<GitHubIcon /> <span className="ml-2">VISIT REPO</span></Button>
	</div>
)

export default AboutPage;