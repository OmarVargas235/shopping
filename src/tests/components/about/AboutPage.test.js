import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AboutPage from '../../../components/about/AboutPage.jsx';

test('Comprobando el marcado html', () => {
	const wrapper = shallow(<AboutPage />);

	expect(toJson(wrapper)).toMatchSnapshot();
});