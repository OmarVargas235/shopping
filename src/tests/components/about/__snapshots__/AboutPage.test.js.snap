// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comprobando el marcado html 1`] = `
<div
  className="text-center d-flex flex-column align-items-center container"
>
  <h2
    className="font-normal"
  >
    ABOUT
  </h2>
  <p
    className="my-4"
  >
    This project was built for practice purposes using Context API by React.
  </p>
  <styled.button
    className="btn d-flex text-center align-items-center"
  >
    <ForwardRef(SvgGitHub) />
     
    <span
      className="ml-2"
    >
      VISIT REPO
    </span>
  </styled.button>
</div>
`;
